scRNA-seq-advanced/04-differential_expression.Rmd

```diff
@@ -12,156 +12,255 @@ output:
 ```
 
-Interestingly, it looks like samples from the ARMS and ERMS subdiagnoses tend to integrate with each other rather than integrate all together. 
+Interestingly, it looks like samples from the ARMS and ERMS subdiagnoses tend to integrate with or samples of the same subdiagnosis rather than integrate all together. 
 
-We can also look at the distribution of cell types to see if these subdiagnoses have different or shared cell types.
+In the integration notebook we also looked at the distribution of cell types after integration.
+We expect that cells of the same cell type should integrate with other cells of the same type.
+Is that the case with this dataset?
 
-*Talk about looking across cell types and comparing cell types* 
+_A word of caution when evaluating the cell type results for this dataset: The cell types for this dataset were assigned in a two stage process.
+The first stage assigned cells as tumor or non-tumor.
+The next stage further classified tumor cells into type of tumor cell (i.e., myoblast, myocyte, or mesoderm).
+Not all samples contain tumor cells that were further classified and samples will either have all `Tumor` or a mix of `Tumor_Mesoderm`, `Tumor_Myoblast`, and `Tumor_Myocyte`._
 
-```{r}
-# facet the UMAP by sample to show the distribution of cell types across each subdiagnosis
+```{r celltype UMAP}
+# # UMAP of all libraries, separating by diagnosis group and labeling cell type
+scater::plotReducedDim(integrated_sce,
+                       dimred = "fastmnn_UMAP",
+                       # color each point by cell type
+                       colour_by = "celltype_broad",
+                       point_size= 0.5, 
+                       point_alpha = 0.4)
+```
+
+Unlike with the previous datasets we have seen where all cells of the same cell type always grouped together, this dataset shows some slightly different patterns.
+We see that there are at least 2 clear distinct groups for each cell type, 1 in each half of the UMAP.
+It looks like this is because the difference in diagnosis group accounts for differences across all cells in the sample, even those from the same type. 
+We can take advantage of another `ggplot` tool, the `facet_wrap()` function to look at two (or more, but we'll start with two) variables in the `colData` at once - the cell type and the subdiagnosis.
+In the below plot we will color our cells by cell type but we will also use `facet_wrap()` to plot the cells from each subdiagnosis in their own separate panel of the plot.
+
+
+```{r celltype subdiagnosis UMAP}
+# # UMAP of all libraries, separating by diagnosis group and labeling cell type
 scater::plotReducedDim(integrated_sce,
                        dimred = "fastmnn_UMAP",
+                       # color each point by cell type
                        colour_by = "celltype_broad",
                        point_size= 0.5, 
                        point_alpha = 0.4,
                        other_fields = "diagnosis_group") +
```

Comment:
Add a comment here that this is needed for `scater` appeasement.
They will have already seen this in the integration notebook but a reminder can't hurt, because it's weird!